Samples/CommandBinding/MauiEdit.WinForms/MauiEdit.WinForms.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+	<EnablePreviewFeatures>true</EnablePreviewFeatures>
+	<OutputType>WinExe</OutputType>
+    <TargetFramework>net7.0-windows</TargetFramework>
+    <UseWindowsForms>true</UseWindowsForms>
+    <LangVersion>10.0</LangVersion>
+	<Nullable>enable</Nullable>
+	<ImplicitUsings>enable</ImplicitUsings>
+  </PropertyGroup>
```

Comment:
Common properties are repeated many times across the solution. Please use Directory.Build.props/targets for those.
Also, consider using the [Central Package Management](https://learn.microsoft.com/nuget/consume-packages/Central-Package-Management).

Our samples should follow the best practices and guidelines, so that our samples help our consumers to learn and embrace those too.